### PR TITLE
Random-no generator in javascript

### PR DESCRIPTION
**Pull request by Premjeet1804**
created a random number generator code in javascript

### CODE REVIEW

**Review by DevTMK — CHANGES_REQUESTED** [review]
Hey @Premjeet1804, you are supposed to create new file using template file.
Take a look at [contributing guidelines](https://github.com/DevTMK/snips/blob/9661c2f0362c46d8383ef5a5a18f0a557052dfde/CONTRIBUTING.md).

**Comment by Premjeet1804** [comment]
ok sir

**Comment by Premjeet1804** [comment]
hi i have done changes as u guided please take a look

**Comment by DevTMK** [comment]
Hi @Premjeet1804, I see unnecessary file changes in this PR, either create a new PR with just the RandomNumber file or update this PR

And follow the template.MD file format for creating new snips (you can refer to [this file](https://github.com/DevTMK/snips/blob/main/src/shuffleArray.md) for formatting and pattern)

File: src/Randomnumber.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+## Snip Langauge
+
+- [JavaScript](#javascript)
+
+### JavaScript
+
+```js
+// Add your snip here
+
+function getRandomArbitrary(min, max) {
```
Comment (DevTMK) [inline src/Randomnumber.md:27]: Just keep one function in your snip, remove other code, and comment

Feel free to let me know if u need any help

Comment (Premjeet1804) [reply src/Randomnumber.md:27]: okay sir Thanks..!

**Comment by Premjeet1804** [comment]
sir i made changes please review them...
and thankyou for your humble support

**Comment by DevTMK** [comment]
> Hi @Premjeet1804, I see unnecessary file changes in this PR, either create a new PR with just the RandomNumber file or update this PR
> 
> And follow the template.MD file format for creating new snips (you can refer to [this file](https://github.com/DevTMK/snips/blob/main/src/shuffleArray.md) for formatting and pattern)

Mate, why did u change ur function logic?
- Do one thing now, create a new Pull request with just the RandomNumber file
- Refer to [this file](https://github.com/DevTMK/snips/blob/main/src/shuffleArray.md) for formatting your new snip file
- And your remember function should return a random number within the provided range 

Feel free to let me know if u need any help

**Comment by Premjeet1804** [comment]
yes sir i'll do that again good-day to you..

**Pull request closed without merging**